.github/workflows/ci.yml

```diff
@@ -23,6 +23,29 @@ env:
   PYTHONIOENCODING: "UTF-8"
 
 jobs:
+  preflight:
```

Comment:
https://github.com/copier-org/copier/blob/600ebdb4920c6c535c6e856d9b3dc504160949e4/tests/test_tools.py#L10

This is already being tested. This part is not needed.